simtools/applications/validate_camera_efficiency.py

```diff
@@ -75,6 +84,34 @@ def _parse(label):
             "Plot the camera efficiency vs wavelength for cherenkov and NSB light."
         ),
     )
+    config.parser.add_argument(
+        "--azimuth_angle",
+        help=(
+            "Telescope pointing direction in azimuth. "
+            "It can be in degrees between 0 and 360 or one of north, south, east or west "
+            "(case insensitive). Note that North is 0 degrees and "
+            "the azimuth grows clockwise, so East is 90 degrees."
+        ),
+        type=CommandLineParser.azimuth_angle,
+        default=0,
+        required=False,
+    )
+    config.parser.add_argument(
+        "--zenith_angle",
+        help="Zenith angle in degrees (between 0 and 180).",
+        type=CommandLineParser.zenith_angle,
+        default=20,
+        required=False,
+    )
+    config.parser.add_argument(
+        "--nsb_spectrum",
+        # FIXME - describe format of the sim_telarray file or better, use a standard
+        # format and convert on the fly to sim_telarray format
+        help="File with NSB spectrum to use (following sim_telarray required format).",
```

Comment:
Both here and in the docstring, it would be good to give more details on the format for non-simtel_array experts. Also, the format in simtel_array could eventually change and then this would not fit here anymore.

Reply:
That's a good point that I was too lazy to deal with before. Now I implemented a function that converts to the sim_telarray format so that the user just has to provide two columns as expected (but still backward compatible). The description was also improved.

Reply:
Thanks. That is good